MSOE.MediaComplete/MainWindow.xaml.cs

```diff
@@ -501,20 +501,25 @@ private void PlaylistTree_SelectionChanged(object sender, RoutedPropertyChangedE
         /// </summary>
         /// <param name="sender"></param>
         /// <param name="e"></param>
-        private void FolderTree_OnSelectionChanged(object sender, EventArgs e)
+        private void FolderTree_OnMouseUp(object sender, EventArgs e)
         {
             if (!IsLoaded)
                 return;
 
-            FormCheck();
-            if (FolderTree.SelectedItems != null)
+            FormCheck(); 
+            var newItems = new List<FolderTreeViewItem>();
+            newItems.AddRange(FolderTree.SelectedItems.Cast<FolderTreeViewItem>());
+            if (FolderTree.SelectedItems != null && FolderTree.SelectedItems.Count >= 0 && !_oldSelectedItems.SequenceEqual(newItems))
             {
+                _oldSelectedItems = newItems;
                 RootLibraryFolderItem.IsSelected = FolderTree.SelectedItems.Count == 0;
                 Songs.View.Refresh();
             }
             ClearDetailPane();
         }
 
+
+        private List<FolderTreeViewItem> _oldSelectedItems = new List<FolderTreeViewItem>();
```

Comment:
If the screen has not loaded and this function tries to run, app crashes

On mouseup, if the items that are currently selected are the same as what was there for the last mouse up, then nothing happens. if they are different, it refreshes the treeview, and resets the 'last mouse up' list of items. 

in any case, clicking on the treeview clears selections from the detail pane